src/Frontend/components/portal/signup/Form.js

```diff
@@ -10,168 +10,98 @@
+ * @param type
+ * @param touched
+ * @param error
+ */
+const customComponent = ({ input, label, type, meta: { touched, error } }) =>
+  <FormGroup>
+    <ControlLabel>{label}</ControlLabel>
+    <FormControl {...input} placeholder={label} type={type} />
+    {touched && ((error && <Alert bsStyle="danger">{error}</Alert>))}
+  </FormGroup>;
 
-  /**
-   * Registers the store.
-   *
-   * @returns {void}
-   */
-  componentDidMount() {
-    connector(userStore).subscribe(this._handleChange);
-  }
+/**
+ * Custom Component builder for selectables
+ *
+ * @param input
+ * @param label
+ */
+const dropDownComponent = ({input, label }) =>
+  <div >
+    <b>{label}</b>
+    <div onChange={(e) => {input.onChange(e.target.value);}}>
+      <input type="radio" value="de" name="locale" /> Deutsch (DE) <br />
+      <input type="radio" value="en" name="locale" defaultChecked="defaultChecked"/> English (EN) <br />
+    </div>
+  </div>;
 
-  /**
-   * Removes the store.
-   *
-   * @returns {void}
-   */
-  componentWillUnmount() {
-    connector(userStore).unsubscribe(this._handleChange);
-  }
-  /**
-   * Renders the component.
-   *
-   * @returns {React.Element} The vDOM markup.
-   */
-  render() {
-    const callback = () => {
-    };
 
-    return (
-      <form onSubmit={this._createAccount.bind(this)}>
-        <Suggestions suggestions={this.state.validation.suggestions} />
-        {this.state.success ? <Success /> : null}
-        <FormField
-          name="username"
-          type="text"
-          value={this.helper.getValue(this.state.data.username, 'username')}
-          autoFocus={true}
-          errors={this.state.validation.errors}
-          helper={this.helper} />
-        <FormField
-          name="password"
-          type="password"
-          value={this.helper.getValue(this.state.data.password, 'password')}
-          errors={this.state.validation.errors}
-          helper={this.helper} />
-        <FormField
-          name="email"
-          type="email"
-          value={this.helper.getValue(this.state.data.email, 'email')}
-          errors={this.state.validation.errors}
-          helper={this.helper} />
-        <SelectableField
-          name="locale"
-          errors={this.state.validation.errors}
-          helper={this.helper}
-          value={this.helper.getValue(this.state.data.locale, 'locale')}
-          options={{ de: 'Deutsch (Deutschland)', en: 'English (USA)' }} />
-        <Recaptcha
-            sitekey={siteKey}
-            render='explicit'
-            onloadCallback={callback}
-            verifyCallback={this.verifyCallback.bind(this)}
-        />
-        <LoadableButtonBar btnLabel={this.helper.getFormFieldAlias('button')} progress={this.state.progress} />
-      </form>
-    );
-  }
+/**
+ * Custom Component for recaptcha
+ *
+ * @param input
+ */
+const recaptchaComponent = ({ input }) =>
+  <Recaptcha
+    sitekey={siteKey}
+    render='explicit'
+    onloadCallback={() => {
+    }}
```

Comment:
do we really need this? According to the `react-recaptcha` code (https://github.com/appleboy/react-recaptcha/blob/master/src/index.js#L7) the `onloadCallback` is an optional parameter.